releasenotes/notes/apm-rename-target-tps-6e7fce5828fcccb4.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+deprecations:
+  - |
+    APM: DD_APM_MAX_TPS config setting is deprecated in favor of the more
```

Comment:
Let's clarify here that the behavior is the same it's just that the name is being changed to more accurately reflect the existing behavior